src/main/docker/jenkins/master.key

```diff
@@ -0,0 +1 @@
+df56f4d532a3ed37c33d2749434a502f3e38eb10fae5d3310bb6d6e5680c98177a7d7e83db43ad8c27c34f7a7d9034565c0c96f6283e79111fcdba3a4a6488839480fb771dd6468f35af67e5adc9504fc633553c2e9aa715df418bc58d6610d65c406ecba5c21de8f2e1c57f02240ad80183c266e450a91965cda325997d14e1
```

Comment:
I would suggest to replace the name of the file to something like `master-key-only-for-testing.key` .. just to make that "more than clear" to anyone :)

Reply:
Good point!